src/main/java/seedu/duke/Main.java

```diff
@@ -0,0 +1,33 @@
+package seedu.duke;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+import seedu.duke.gui.MainWindow;
+
```

Comment:
Do you need to separate imports of different packages by a blank line here? Correct me if I'm wrong